Dockerfile.node

```diff
@@ -0,0 +1,28 @@
+FROM node:12-alpine
```

Comment:
Should this be tied to a specific version of alpine say 3.10? Otherwise we tend to have problems with dependencies